packages/core/src/types/styleASTTypes.ts

```diff
@@ -0,0 +1,189 @@
+//#region Style AST
+// TODO: unify type name convention (e.g. stop using `I` for interfaces and drop some of the Haskell ported types)
+
+import { ASTNode, Identifier } from "./ASTTypes";
+import { StyVar, BindingForm, Expr, Path } from "./shapeEvalTypes";
```

Comment:
These should all be defined in this file instead

Reply:
these meaning https://github.com/penrose/penrose/pull/526#discussion_r592051644 ?